core/src/main/java/com/infinum/jsonapix/core/adapters/TypeAdapter.kt

```diff
@@ -4,7 +4,7 @@ interface TypeAdapter<Model> {
     fun rootLinks(): String = "links"
     fun resourceObjectLinks(): String = "links"
     fun relationshipsLinks(): String = "links"
-    fun errors(): String = "errors"
+    fun errors(): String = "error"
```

Comment:
`errors` is more fitting in my opinion, because it indicates to the `errors` array from the Json API spec. 
As for the value of it being "error" by default, this is not that important, but I decided to go with singular form because it's a discriminator for one class -> `DefaultError`